obs_common/service_status.py

```diff
@@ -9,63 +9,22 @@
 how far behind different server environments are from main tip.
 """
 
-import argparse
 import json
-import os
 import sys
 from urllib.parse import urlparse
 from urllib.request import urlopen
+from pathlib import Path
 
+import click
+import tomllib
```

Comment:
I typically run this script from my local machine--not in a container. This changes the script to need to be run inside the container.

We might want to add a `make servicestatus` rule to services to reduce the number of steps it takes to do something we'll do semi-frequently especially during support rotations when we're trying to figure out whether to do a deploy.

Reply:
I remember vaguely that we talked about pushing obs-common to our private package index so we can `pipx install` it. Is this still the plan? If so, external dependencies should be fine.

Reply:
+1 for make servicestatus